Oppgave 2.ipynb

```diff
@@ -0,0 +1,1293 @@
+{

```

Comment:
<p>Litt overraskende resultat at ikke RF-modellen presterte bedre for deg, men jeg kan ikke se noen åpenbare feil her</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bsingstad/DMBA2024/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>